app/src/main/java/com/portfolio/tasky/entry/usecases/domain/EmailPatternValidator.kt

```diff
@@ -1,4 +1,4 @@
-package com.portfolio.tasky.entry.usecases
+package com.portfolio.tasky.entry.usecases.domain
```

Comment:
Usually domain has usecases, not the other way around

So the package should be:
entry -> Domain -> Usecases